src/lib/utils/logHandler.js

```diff
@@ -55,9 +55,13 @@ class LogHandler {
       processedMessages = [msg.message];
     } else if (Array.isArray(msg.message)) {
       msg.message.forEach(message => {
-        if (Array.isArray(message)) message = message.join('\n');
-        let lines = message.split("\n");
-        lines.forEach(line => processedMessages.push(line));
+        if (Array.isArray(message)) {
+          message = message.join('\n');
+        }
+        if (typeof message === 'object') {
+          return processedMessages.push(JSON.stringify(msg.message));
+        }
+        message.split("\n").forEach(line => processedMessages.push(line));
```

Comment:
Could line 64 replace line 59?

Reply:
In other words:
```
if (Array.isArray(message)) {
  message.forEach(line => processedMessages.push(line));
}
```

Reply:
Yeah, I think I can improve it. I'll push another commit

Reply:
If `line` could actually be lines that we want split on `'\n'`, could use an inner `forEach`:
```
message.forEach(line => line.split('\n').forEach(line => processedMessages.push(line)));
```

Reply:
I tried doing a recursive function for arrays of messages but it only ended up breaking the message coloring, so I ended just adding your suggestion.